tool/scaffold/src/main/java/org/openscience/cdk/tools/scaffold/NetworkNode.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2022 Julian Zander <zanderjulian@gmx.de>
```

Comment:
Thanks!

Reply:
For putting the copyright notice? Welcome!